core/src/main/scala/kafka/zookeeper/ZooKeeperClient.scala

```diff
@@ -528,7 +528,7 @@ case class SetAclRequest(path: String, acl: Seq[ACL], version: Int, ctx: Option[
   type Response = SetAclResponse
 }
 
-case class GetChildrenRequest(path: String, ctx: Option[Any] = None) extends AsyncRequest {
+case class GetChildrenRequest(path: String, registerWatch: Boolean = true, ctx: Option[Any] = None) extends AsyncRequest {
```

Comment:
I would remove the default since it has serious implications. It should be chosen explicitly.

Reply:
That's a fair point. Let me remove it.